lib/board.rb

```diff
@@ -32,8 +37,45 @@ def free_positions
     flat_board.each_index.select { |index| empty?(flat_board[index]) }
   end
 
+  def content
+    board_string = ""
+
+    board.flatten.each.with_index do |value, index|
```

Comment:
That's quite a long method. Can you think of ways to reduce it? Maybe there's an `Enumerable` way of doing it?